entrypoint.sh

```diff
@@ -40,20 +42,15 @@ DATADIR="/home/erigon-gnosis/.local/share"
 exec erigon --datadir=${DATADIR} \
     --chain=gnosis \
     --http.addr=0.0.0.0 \
-    --http.port=8545 \
```

Comment:
Are we exposing this through the proxy somehow?

Reply:
Its exposed by default, correct me if Im wrong 
https://github.com/ledgerwatch/erigon?tab=readme-ov-file#default-ports-and-firewalls

Reply:
That's not exactly how it works, but it's handled in the Dappnode config file in `exposable` and it's fine to remove the line and keep the default port indeed. Please copy that part from the main file as well: https://github.com/dappnode/DAppNodePackage-erigon/blob/main/dappnode_package.json#L35-L48.